Formula/antidote.rb

```diff
@@ -0,0 +1,34 @@
+class Antidote < Formula
+  desc "Plugin manager for zsh, inspired by antigen and antibody"
+  homepage "https://getantidote.github.io/"
+  url "https://github.com/mattmc3/antidote/archive/refs/tags/v1.6.4.tar.gz"
+  sha256 "336ea89aab33d07d5d2eaa14af2f015d7da657837f64d3e76fa5c54645210950"
+  license "MIT"
+
+  uses_from_macos "zsh"
+
+  def install
+    pkgshare.install "antidote.zsh"
+    pkgshare.install "functions"
+    rm_f Dir["man/*.md"]
+    man.install Dir["man/*"]
+  end
+
+  def caveats
+    <<~EOS
+      To activate antidote, add the following to your ~/.zshrc:
+        source #{pkgshare}/antidote.zsh
+    EOS
+  end
+
+  test do
+    (testpath/".zshrc").write <<~EOS
+      export GIT_TERMINAL_PROMPT=0\n
+      export ANTIDOTE_HOME=~/.zplugins\n
+      source #{pkgshare}/antidote.zsh\n
```

Comment:
```suggestion
      export GIT_TERMINAL_PROMPT=0

      export ANTIDOTE_HOME=~/.zplugins

      source #{pkgshare}/antidote.zsh
```
If you need the extra newlines (why?), you don't have to encode the `\n`. Otherwise,
```suggestion
      export GIT_TERMINAL_PROMPT=0
      export ANTIDOTE_HOME=~/.zplugins
      source #{pkgshare}/antidote.zsh
```

Reply:
I didn't need the extra newlines. I failed homebrew CI with a single line that was too long in a prior commit, so I broke up the string into 3 lines and didn't think to remove the \n.